History.md

```diff
@@ -1,5 +1,18 @@
 ## vNEXT
 
+## v0.7.0
```

Comment:
I think we should bump to v1.

Reply:
@mizzao what do you think of pushing this to 1.0.0?

Reply:
I think you two should own/decide this - I don't really have much of an opinion.

Reply:
My original idea was to merge #121 and #123 and release that under the 0.x  branch and since this is a major rework have that as v1 to have a clear split. Also I recently updated one of my packages and noticed some things which I think also support bumping the version (see my comments bellow on `package.js`).

Reply:
@wreiske Thoughts? Given the amount of changes I would like to push this to v1, which will force people to pay attention during upgrade and allow them to go back in case we missed something.